python/kserve/docs/V1alpha1ClusterServingRuntime.md

```diff
@@ -2,13 +2,13 @@
 
 ClusterServingRuntime is the Schema for the servingruntimes API
 ## Properties
-Name | Type | Description | Notes
------------- | ------------- | ------------- | -------------
-**api_version** | **str** | APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources | [optional] 
-**kind** | **str** | Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds | [optional] 
+Name | Type                                                                                                        | Description | Notes
+------------ |-------------------------------------------------------------------------------------------------------------| ------------- | -------------
+**api_version** | **str**                                                                                                     | APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources | [optional] 
+**kind** | **str**                                                                                                     | Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds | [optional] 
 **metadata** | [**V1ObjectMeta**](https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/V1ObjectMeta.md) |  | [optional] 
-**spec** | [**V1alpha1ServingRuntimeSpec**](V1alpha1ServingRuntimeSpec.md) |  | [optional] 
-**status** | [**object**](.md) | ServingRuntimeStatus defines the observed state of ServingRuntime | [optional] 
+**spec** | [**V1alpha1ServingRuntimeSpec**](V1alpha1ServingRuntimeSpec.md)                                             |  | [optional] 
+**status** | [**V1alpha1ServingRuntimeStatus**](V1alpha1ServingRuntimeStatus.md)                                                                     | ServingRuntimeStatus defines the observed state of ServingRuntime | [optional] 
```

Comment:
These are generated from swagger.json file, so we might need to fix from the source there.

Reply:
No, that was one of the links I could not resolve, like the other 13 listed in the PR description that I need help with :-)